tst/EnergyPlus/unit/AirLoopHVACDOAS.unit.cc

```diff
@@ -4132,7 +4131,8 @@ TEST_F(EnergyPlusFixture, AirLoopHVACDOASTest)
 
     state->dataEnvrn->OutBaroPress = 101325.0;
 
-    state->dataScheduleMgr->Schedule(1).CurrentValue = 1.0; // set availability and fan schedule to 1
+    auto *sched = Sched::GetSchedule(*state, "ALWAYS_ON");
```

Comment:
Don't rely on position to manipulate schedules created by IDF snippets, use the schedule name. It's a little slower, but this is a unit test so speed doesn't matter and it's much more robust and transparent. 

All unit test change are of this form.